app/features/ai_resistant_assignment_generator/tests/test_core.py

```diff
@@ -0,0 +1,16 @@
+# tests/test_core.py
+import pytest
+from app.features.ai_resistant_assignment_generator.core import executor
+
+def test_executor_valid_input():
+    files = ["file1.pdf", "file2.pdf"]
```

Comment:
These test files appear to be from your local directory. For more effective testing, you should use online files accessible through URLs. Your local files will not be accessible in production or to other contributors.